### PR TITLE
updated hero:

### PR DESCRIPTION
**Pull request by jfsgomes**
- added new links to Nav bar (experts still missing)
- reduced margin between title and logo
- logo height and width (web and mobile)

### CODE REVIEW

**Comment by pm-araujo** [comment]
:+1:

File: app/styles/components/_logo.scss [hunk 1]
```diff
@@ -1,7 +1,7 @@
-$logo-width: 179px;
-$logo-height: 120px;
+$logo-width: 149px;
+$logo-height: 100px;
 
-$logo-width-mobile: 76px;
+$logo-width-mobile: 75px;
```
Comment (frm) [inline app/styles/components/_logo.scss:4]: I know you changed these values to match with the ones on Zeplin, but checking the git history, these values were there from the start. In fact, they match the image dimensions.

They're probably a result of some design changes that weren't reflected on Zeplin, so I'd wait for a feedback from @fcbaila

**Comment by frm** [comment]
The nav started breaking on mobile.

screen shot 2016-08-23 at 15 20 05

You'll probably have to change the breakpoints to fix it, or maybe even add a specific breakpoint for the nav to turn mobile. I'm more fond of the latter but whichever one results in better design. /cc @fcbaila

File: app/index.html [hunk 2]
```diff
@@ -33,11 +33,13 @@
       <div class="Nav-links">
         <a href="#mission" class="Nav-link">Mission</a>
         <a href="#services" class="Nav-link">What we offer</a>
+        <a href="#newsletter" class="Nav-link">get involved</a>
         <a href="#officers" class="Nav-link">officers</a>
         <a href="#board" class="Nav-link">board</a>
-        <!-- <a href="#" class="Nav-link">experts</a> -->
+        <a href="#sponsors" class="Nav-link">sponsors and partners</a>
+        <a href="#" class="Nav-link">experts</a>
```
Comment (frm) [inline app/index.html]: While we don't have this section online, let's keep it off the nav.

File: app/styles/theme.scss [hunk 3]
```diff
@@ -16,6 +16,7 @@ $Theme-opacity-white: 0.7;
 $Theme-opacity-black: 0.6;
 
 $Theme-break-mobile: 699px;
+$Theme-break-mobile-nav: 1167px;
```
Comment (frm) [inline app/styles/theme.scss]: As we discussed in person, this makes more sense to move to the `_nav.scss` file and rename it to something like `$nav-break-mobile`

File: app/styles/utilities/_hide.scss [hunk 4]
```diff
@@ -9,3 +9,9 @@
     display: none;
   }
 }
+
+@media (min-width: $Theme-break-mobile-nav) {
```
Comment (frm) [inline app/styles/utilities/_hide.scss]: As we discussed in person, this media query should be related to the `.Nav-burger` component.

File: app/index.html [hunk 5]
```diff
@@ -33,11 +33,13 @@
       <div class="Nav-links">
         <a href="#mission" class="Nav-link">Mission</a>
         <a href="#services" class="Nav-link">What we offer</a>
+        <a href="#newsletter" class="Nav-link">get involved</a>
         <a href="#officers" class="Nav-link">officers</a>
         <a href="#board" class="Nav-link">board</a>
-        <!-- <a href="#" class="Nav-link">experts</a> -->
+        <a href="#sponsors" class="Nav-link">sponsors and partners</a>
```
Comment (frm) [inline app/index.html]: As we discussed with @fcbaila, this link should be removed, as well as the `Mission` link.

File: app/styles/components/_logo.scss (hunk 1 above)
Comment (jfsgomes) [reply app/styles/components/_logo.scss:4]: Confirmed with @fcbaila that the changes in the image dimensions should be done.

File: app/index.html (hunk 2 above)
Comment (jfsgomes) [reply app/index.html]: :+1:

File: app/styles/theme.scss (hunk 3 above)
Comment (jfsgomes) [reply app/styles/theme.scss]: :+1:

File: app/styles/utilities/_hide.scss (hunk 4 above)
Comment (jfsgomes) [reply app/styles/utilities/_hide.scss]: :+1:

File: app/index.html (hunk 5 above)
Comment (jfsgomes) [reply app/index.html]: :+1:

File: app/styles/components/_nav.scss [hunk 6]
```diff
@@ -116,3 +118,9 @@ $nav-burger-height: 16px;
     display: none;
   }
 }
+
+@media (min-width: $nav-break-mobile+1) {
```
Comment (frm) [inline app/styles/components/_nav.scss]: Why the `+1` here?

Comment (jfsgomes) [reply app/styles/components/_nav.scss]: I noticed that when the window width is equal to $nav-break-mobile the burger disappears. This way it behaves correctly.
screenshots

Comment (jfsgomes) [inline app/styles/components/_nav.scss]: I double checked it today and it looks fine without the +1, so I'll remove it

Comment (jfsgomes) [reply app/styles/components/_nav.scss]: As discussed in person, I fixed the media query to avoid the use of the +1

**Comment by frm** [comment]
👍

**Pull request merged**